platform/example.anchor-config.yaml

```
@@ -0,0 +1,100 @@
+stellar:
+  anchor:
+    # Configure the application from this file after it is loaded in memory
+    # If `config` is in-memory, the yaml file contains all settings for the server.
+    config: in-memory
+    #
+    # Configures how the applications read the
+    #
+    app-config:
+      type: config-spring-property                # Activate [config-spring-property] module
+      settings: app-config                        # The location of the configuration data
+    data-access:
+      type: data-spring-jdbc                      # Activate [config-spring-jdbc] module.
+      settings: data-spring-jdbc-settings         # The location of the configuration data in this file.
+    logging:
+      type: logging-logback
+      settings: logging-logback-settings
+
+#
+# Application settings
+#
+app-config:
+  # general
+  app:
+    stellarNetworkPassPhrase: Test SDF Network ; September 2015
```

nit: `passphrase` is usually a single word, like in this json response: https://horizon-testnet.stellar.org/

```suggestion
    stellarNetworkPassphrase: Test SDF Network ; September 2015
```

---

Changed.